armsrc/iso14443a.c

```diff
@@ -2535,6 +2541,11 @@ void Mifare1ksim(uint8_t flags, uint8_t exitAfterNReads, uint8_t arg2, uint8_t *
 			LED_C_OFF();
 			crypto1_destroy(pcs);
 			cardAUTHKEY = 0xff;
+			if (flags & FLAG_RANDOM_NONCE) {
+				nonce = prand();
+			} else {
+				nonce++;
```

Comment:
Wouldn't this break the standard reader attack? As far as I remember it is based on two ar/nr pairs for the same nt. Only Moebius attack needs varying nt.